pkg/segment/memory/limit/memorylimit.go

```diff
@@ -52,7 +54,7 @@ func InitMemoryLimiter() {
 	memory.GlobalMemoryTracker = &structs.MemoryTracker{
 		TotalAllocatableBytes:   totalAvailableSizeBytes,
 		RotatedCMIBytesInMemory: rotatedCMIBytes,
-		SegSearchRequestedBytes: segSearchBytes,
+		SegSearchRequestedBytes: AllocatedSegSearchBytes,
```

Comment:
If the first one is called SegSearchRequestedBytes then we should call this SegSearchAllocatedBytes